lib/github.com/mpllang/mpllib/CheckSort.sml

```diff
@@ -36,17 +36,10 @@ struct
     let
       val n = Seq.length input
       val input' = Seq.mapIdx (fn (i, x) => (i, x)) input
-      fun compare' ((i1, k1), (i2, k2)) = compare (k1, k2)
+      fun compare' ((_, k1), (_, k2)) = compare (k1, k2)
 
       val result = sort_func compare' input'
 
-      val noElemsMissing: bool =
-        DS.reduce (fn (a, b) => a andalso b) true
-          (DS.inject
-            ( DS.tabulate (fn _ => false) n
-            , DS.map (fn (i, k) => (i, true)) (DS.fromArraySeq result)
-            ))
-
```

Comment:
We should leave this in; the fact that it's there makes me want to revisit this structure and check if we did something wrong.